sensor/kubernetes/complianceoperator/handler_impl.go

```diff
@@ -39,6 +46,8 @@ type handlerImpl struct {
 	stopSignal        concurrency.Signal
 	started           *atomic.Bool
 	complianceIsReady *concurrency.Signal
+	numRetries        int
+	callTimeout       time.Duration
```

Comment:
nitpicking: Just a consideration. I'm thinking, is there a better way to indicate in the "variable" name that this is a global timeout, not per retry? Right now, it's not so clear.

Reply:
Done. Let me know if it's still not clear.